validator/tests/test_scheduler/data/complex_dependency_failures.yaml

```diff
@@ -0,0 +1,551 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+---
+-  #------------------------- Batch 1
```

Comment:
Are there any notes about these batches that can make it more informative about what each one is testing, or if they are all related to one test?

Reply:
Yes, I could add notes at the top of each yaml file that explain the emphasis of each test.

Reply:
Do you have thoughts on how to be more informative?

Reply:
At least highlight where the poison pill is - i.e. which batch/txn is bad in the group.

Reply:
Yeah, I will put more notes in the yaml about what is tricky or hard for the scheduler.